dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -279,6 +279,21 @@ public void testLinkageErrorsUnderProvidedDependency() throws IOException, Parsi
     }
   }
 
+  @Test
+  public void testZeroLinkageErrorShowsZero() throws IOException, ParsingException {
+    // grpc-auth does not have a linkage error, and it should show zero in the section
+    Path zeroLinkageErrorHtml = outputDirectory.resolve("io.grpc_grpc-auth_1.17.1.html");
+    Assert.assertTrue(Files.isRegularFile(zeroLinkageErrorHtml));
+
+    try (InputStream source = Files.newInputStream(zeroLinkageErrorHtml)) {
+      Document document = builder.build(source);
+      Nodes staticLinkageTotalMessage = document.query("//p[@id='static-linkage-errors-total']");
+      Truth.assertThat(staticLinkageTotalMessage.size()).isEqualTo(1);
```

Comment:
Up to you, but in simple cases like this I'm not sure Truth adds anything over an simple JUnit assert

Reply:
My preference is Truth because I always get confused the order of _expected_ and _actual_ arguments in JUnit asserts.